requirements.txt

```diff
@@ -11,6 +11,8 @@ django-heroku==0.3.1
 django-js-asset==1.2.2
 djangorestframework==3.12.4
 djangorestframework-simplejwt==4.6.0
+dnspython==2.1.0
```

Comment:
what is dnspython used for?  don't we only need `email-validator` to validate an email?

Reply:
Hi @codyseibert this dnspython is required for email-validator package and is install automatically. It's just a DNS toolkit